src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -128,8 +139,18 @@ public void TestEnvironmentProperty()
             //Exception not thrown with invalid key
             Assert.Throws<ArgumentNullException>(() => environment.Add(null, "NewValue2"));
 
-            //Invalid Key to add
-            Assert.Throws<ArgumentException>(() => environment.Add("NewKey2", "NewValue2"));
+            // Invalid Key to add
```

Comment:
"Exception not thrown with invalid key" above should say "Exception thrown with invalid key"? although hardly worth a comment